src/Training.AirBnb.Clone.Backend/AirBnb.Api/Controllers/NotificationsController.cs

```diff
@@ -11,17 +11,17 @@ public class NotificationsController : ControllerBase
     {
         private readonly IEntityBaseService<EmailTemplate> _emailTemplateService;
         private readonly IEntityBaseService<Email> _emailService;
-        private readonly IEmailMenegmentService _emailMenagmentService;
+        private readonly IEmailManagementService _emailMenagmentService;
 
-        public NotificationsController(IEntityBaseService<EmailTemplate> entityBaseService, IEntityBaseService<Email> emailService, IEmailMenegmentService emailManagementService)
+        public NotificationsController(IEntityBaseService<EmailTemplate> entityBaseService, IEntityBaseService<Email> emailService, IEmailManagementService emailManagementService)
         {
             _emailTemplateService = entityBaseService;
             _emailService = emailService;
             _emailMenagmentService = emailManagementService;
         }
 
-        [HttpPost("emailMenagment")]
-        public async Task<IActionResult> SendEmail([FromRoute] Guid userId, [FromRoute] Guid templateId)
+        [HttpPost("emailManagement")]
```

Comment:
bu yerda endpoint [HttpPost("{userId:guid}/{tempateId:guid}")] bunaqa bo'lsa yaxshimasmi, EmailManagement degan model yo'qku

Reply:
To'g'ri. Api Route da email send qilish uchun ekanligi bilinib turishi kerak chunki method nomi api route da ko'rinmaydi. Agar emailManagement bo'ladigan bo'lsa bu api dan foydalanayotgan client nima vazifa bajarishini tushunmay qoladi.